.github/workflows/CI.yaml

```diff
@@ -26,14 +26,23 @@ jobs:
         os: [macOS-latest, ubuntu-latest, windows-latest]
         python-version: ["2.7", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10"]
         exclude:
-        - os: macOS-latest
-          python-version: 3.5	
         - os: macOS-latest
           python-version: 3.6
         - os: macOS-latest
           python-version: 3.7
         - os: macOS-latest
           python-version: 3.8
+        - os: macOS-latest
```

Comment:
Why do we want to exclude all these versions? If we don't want to run on OSX and windows, we could just do
`os: [ubuntu-latest, ]`

Reply:
To cut down on CI time and resources I only test lowest and highest Python on the windows and macOS runners.

Reply:
I see the logic now. Might be good to add a comment to explain that. Otherwise, I'm happy with this PR.